lib/abr/simple_abr_manager.js

```diff
@@ -32,6 +33,7 @@ goog.require('shaka.util.Timer');
  * </p>
  *
  * @implements {shaka.extern.AbrManager}
+ * @implements {shaka.util.IDestroyable}
```

Comment:
Would it work to instead say that `shaka.extern.AbrManager` implements `shaka.util.IDestroyable`?

Reply:
I tried to do so, but files in externs won't let me import the interface using goog.require. I did like the other externs: write the destroy method in the extern, and add the implements on the class 
```javascript
goog.require('shaka.util.IDestroyable');
/**
 * @externs
 */
/**
 * @interface
 * @implements {shaka.util.IDestroyable}
 * @exportDoc
 */
shaka.extern.AbrManager = class {}
```

```
/Users/vincentvalot/Projects/Personal/shaka-player/externs/shaka/abr_manager.js:7:0: ERROR - [JSC_LATE_PROVIDE_ERROR] Required namespace "shaka.util.IDestroyable" not provided yet.
  7| goog.require('shaka.util.IDestroyable');
     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

/Users/vincentvalot/Projects/Personal/shaka-player/externs/shaka/abr_manager.js:7:0: ERROR - [JSC_UNDEFINED_EXTERN_VAR_ERROR] name goog is not defined in the externs.
  7| goog.require('shaka.util.IDestroyable');
     ^^^^

2 error(s), 0 warning(s)
[ERROR] Build failed

```

Reply:
Fair enough.